### PR TITLE
Add Fujicoin

### PR DESCRIPTION
**Pull request by fujicoin**
Please add Fujicoin.

### CODE REVIEW

**Comment by martinboehm** [comment]
Hi, your PR contains only the config file. See for example [this](https://github.com/trezor/blockbook/pull/79) PR as a reference of coin implementation.
Also, if possible, please use ports xx48, the ports xx47 is already used by another coin.

**Comment by fujicoin** [comment]
Thank you for your comment.
I added the necessary files and changed the port number to xx48.
By the way, how should I make testTxPacked1?
https://github.com/fujicoin/blockbook/blob/07300c6b2c3eb8ccdfc938e921d3bd260bb538a1/bchain/coins/fujicoin/fujicoinparser_test.go#L84

**Comment by fujicoin** [comment]
screenshot_2019-01-13 trezor fujicoin explorer

Deployment in manual build on test environment (ubuntu 16.04) is completed. Function and various display contents are normal. The broadcast of the transaction is also normal.

**Comment by fujicoin** [comment]
Deployment in production environment is completed.
The URL is as follows:
https://explorer.fujicoin.org/

**Comment by fujicoin** [comment]
screenshot_2019-01-14 trezor wallet

However, using 'https://explorer.fujicoin.org/' will not connect from the TREZOR wallet.
'https: // 192.168.xx.xx: 9148' is used to connect from the TREZOR wallet.
Is something wrong with the setting of apache2?

**Comment by fujicoin** [comment]
screenshot_2019-01-14 trezor wallet 1

The third point is settled.
Added the following to the proxy setting of apache2.
`
ProxyPass /socket.io/ wss://localhost:9148/socket.io/
ProxyPassReverse /socket.io/ wss://localhost:9148/socket.io/
`

**Comment by fujicoin** [comment]
point 1:
I added fujicoin-releases.asc.
point 2:
I am not a go programmer, so I do not know how to pack.

**Comment by fujicoin** [comment]
screenshot_2019-01-14 trezor wallet 2

I used a TREZOR wallet to test for deposits and withdrawals. The result is good.

**Comment by martinboehm** [comment]
Packing: use this string as packedTx and the test will pass:
`0023f3648bc2d3dd5c0100000001ec5666d4e842926c6e872fd7f12a87183b298464716dec1b03fc956777e06a70010000006a473044022062949ffe184af6bdc8d77e07c2caf1bd4e36d977bc27e24ab6d24c9e735d76b802201f42950f6fa005126f1fbd5cb9d6f0838d94adb70a05517b3eebd144a0ec2fd6012102534f95a0a13bbc3e261f966edd057a1dc9819a4e6bb21d81a633d5500d184a7bfeffffff0200e876481700000017a9146b8a69f8b43bc06b886c0668c93197be5152e8598740494d552e0000001976a914df783f980b38d7a64cee23202af95609b547fde888ac62f32300`

File: tests/sync/testdata/fujicoin.json [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+                            {
+                                "value": 486.80545257,
+                                "n": 5,
+                                "scriptPubKey": {
+                                    "hex": "a9149bc786641a703416b43429ffe6879b47d78f00f787"
+                                }
+                            },
+                            {
+                                "value": 385.54800875,
+                                "n": 6
```
Comment (wakiyamap) [inline tests/sync/testdata/fujicoin.json]:
```suggestion
                                "n": 6,
```

**Comment by fujicoin** [comment]
@martinboehm @wakiyamap tnx, I fixed it.

**Comment by martinboehm** [comment]
I am able to run Fujicoin Blockbook, however I cannot merge your PR as the integration tests are failing. What problems I see (there may be more):
1) In the `tests/rpc/testdata/fujicoin.json` are wrong txids in the `blockTxs`
2) In the `tests/sync/testdata/fujicoin.json` are wrong height numbers in the `fakeBlocks` and `realBlocks `

Please make the integration tests working (see the [test documentation](https://github.com/trezor/blockbook/blob/master/docs/testing.md) for more info).

**Comment by fujicoin** [comment]
I fixed the pointed out part. I tried testing myself to find out more bugs, but it does not work. I'm sorry but please run the test again.

**Comment by fujicoin** [comment]
Finally I can run the test. I corrected the test files and passed all the tests.

**Comment by fujicoin** [comment]
@martinboehm Thank you for your kind support. I think that this repository is wonderful. I pray for the development of TREZOR.

**Pull request merged**